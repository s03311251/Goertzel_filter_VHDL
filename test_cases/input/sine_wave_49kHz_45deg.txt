36A0
3C6B
3F89
3FB1
3CDD
3753
2F97
2663
1C96
131B
0AD6
048F
00DC
0017
0252
0758
0EAF
17A7
2168
2B07
339C
3A59
3E9C
3FFE
3E5E
39E3
32F8
2A45
209A
16E1
0E03
06D7
0208
000A
010E
04FB
0B72
13D9
1D63
272D
3049
37DE
3D34
3FCB
3F64
3C0A
360C
2DFC
249B
1ACB
1179
0984
03AC
007E
0047
030C
0889
103B
1969
2336
2CB5
3502
3B55
3F17
3FEB
3DC0
38C8
317B
288A
1ECB
1529
0C8C
05C3
0172
0000
0191
05FE
0CDE
158A
1F32
28ED
31D1
3908
3DE5
3FF2
3EFE
3B1F
34B4
2C56
22CF
1905
0FE2
0844
02E0
003A
0091
03DD
09CD
11D5
1B31
