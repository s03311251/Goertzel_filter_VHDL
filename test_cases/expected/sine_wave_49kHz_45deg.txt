FFEE1C20
FFEC2E20
